README.md

```diff
@@ -49,25 +49,30 @@ for (var mealName in meals) {
 
 Much better.
 
-**NOTE**: You can use `for...in` loops with arrays, too, but the variable that you get will just be the index (in order), so this particular loop is _usually_ used with objects.
+***NOTE***: You can use `for...in` loops with arrays, too, but the variable that you get will just be the index (in order), so this particular loop is _usually_ used with objects.
 
 ## Instructions
-+ We've given you a function `setCart()` which takes one argument, an array, and sets `cart` (a variable that we've provided) to that array.
-
-+ We've also given you a function `total` which does not accept any arguments. It iterates over the items in `cart` and adds up their prices, then returns the total.
-
-+ Define a global variable (use `var` at the top level) called `cart`. Initialize it as an empty array.
-
-+ Define a function `getCart` that takes no arguments and returns the `cart`.
-
-+ Define a function `addToCart`. This function should accept one argument, the item the user wants to purchase.
-    This function should automatically set a price for this item by generating a random number between 0 and 100. (Hint: `Math.random()` generates a random number in [0, 1] (0 inclusive, 1 non-inclusive); `Math.floor()` rounds a number down to the nearest integer.)
-    This function should add the item and the price as an object (`{item: price}`) to the `cart` array. This function should print out to the console `<item> has been added to your cart.` and return the cart.
-
-+ Define a function `viewCart` which does not accept any arguments. This function should loop over every item in `cart` to print out `"In your cart you have [item and price pairs]."`. If there isn't anything in your cart, the function should print out `"Your shopping cart is empty."`.
-
-+ Define a function `removeFromCart` which accepts one argument, the name of the item you wish to remove. If the item isn't in the cart, the function should print out `"That item is not in your cart."`. If the item is in your cart, it should remove the object from the `cart` array. Then return the cart. (**HINT**: Check each object's key to see if it matches the parameter, then remove it if it matches. You might find [hasOwnProperty](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty) to be useful.)
-
-+ Define a function `placeOrder` which accepts one argument, a credit card number. If no argument is received, then the function should print out `"We don't have a credit card on file for you to place your order."`. If there is a credit card on file, the function should print out `"Your total cost is $${total()}, which will be charged to the card ${cardNumber}."`. The function should empty the `cart` array.
++ We've started you off with a `cart` variable that points at an empty array.
```

Comment:
Look at how this entire section gets rendered. It's kinda hard to read. Maybe reduce on level of bulleting so the initial is just 1,2,3?